DEHPEcosimPro.Tests/ViewModel/Rows/VariableRowViewModelTestFixture.cs

```diff
@@ -95,21 +94,21 @@ public void VerifyAverageCalculation()
                 DisplayName = new LocalizedText("", "DummyVariable0")
             }, new DataValue() { Value = .2 }));
 
-            var newValues = new List<(object Value, DateTime When)>()
+            var newValues = new List<TimeTaggedValueRowViewModel>()
             {
-                (131234, DateTime.MinValue.AddDays(1)), 
-                (-143298.5224323, DateTime.MinValue.AddDays(1)),
-                (2u, DateTime.MinValue.AddDays(1)),
-                (44.87613, DateTime.MinValue.AddDays(1)),
-                (0.42e2, DateTime.MinValue.AddDays(1)),
-                (.12387, DateTime.MinValue.AddDays(1)),
-                (2ul, DateTime.MinValue.AddDays(1))
+                new TimeTaggedValueRowViewModel(131234, DateTime.MinValue.AddDays(1)), 
```

Comment:
This is I guess placeholder for when SampledFunctionParameterType makes its appearance.

Reply:
Yes indeed